proc_scripts/CPMG_phasing.py

```diff
@@ -2,7 +2,9 @@
 from sympy import symbols
 from proc_scripts import *
 import math
-def center_CPMG_echo(s, axis='t2',fl=None):
+import numpy as np
+logger=init_logging('info')
+def find_echo_center(s, axis='t2',fl=None):
```

Comment:
so this function offers nothing new, and should not exist.

you might want to copy the documentation over to hermitian function test, if it doesn't have any.